Release 0.1.3 (#22)
memgraph/helm-charts: charts/memgraph-lab/README.md

| `podAnnotations`             | Annotations to add to the pod                                                                           | `{}`                                   |
| `resources`                  | CPU/Memory resource requests/limits. Left empty by default.                                             | `{}` (See note on uncommenting)        |
| `serviceAccount.create`      | Specifies whether a service account should be created                                                   | `true`                                 |
| `serviceAccount.annotations` | Annotations to add to the service account                                                               | `{}`                                   |
| `serviceAccount.name`        | The name of the service account to use. If not set and create is true, a name is generated.             | `""`                                   |

Memgraph Lab can be further configured with environment variables in your `values.yaml` file.

```yaml
env:
  - name: QUICK_CONNECT_MG_HOST
    value: memgraph
  - name: QUICK_CONNECT_MG_PORT
    value: "7687"
```

> Check all supported environment variables from the Memgraph website, [Data Visualization Section](https://memgraph.com/docs/data-visualization/install-and-connect#environment-variables)

Reviewer: ```suggestion
Check all supported environment variables on [Memgraph Lab documentation](https://memgraph.com/docs/data-visualization/install-and-connect#environment-variables).
```
